pageStyles/constants.scss

```diff
@@ -0,0 +1,64 @@
+﻿
+//Site Colours
+$softwireBlack: rgb(24, 24, 24);
+$softwireYellow: rgb(255, 193, 0);
+$softwireWhite: rgb(255, 255, 255);
+
+//Fonts
+$softwireFont: Open Sans, sans-serif;
+$desktopHeaderFontSize: 20px;
+$desktopSubHeaderFontSize: 12px;
+$buttonFontSize: 9px;
+@mixin LargeHeaderYellow() {
+  color: $softwireYellow;
+  font-size: $desktopHeaderFontSize;
+  justify-content: flex-end;
+}
+@mixin MediumWhiteHeader() {
+  color: $softwireWhite;
+  font-size: $desktopSubHeaderFontSize;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  padding: 2% 2% 1% 2%;
+ 
+}
+@mixin Instructions() {
+  color: $softwireWhite;
+  font-size: $buttonFontSize;
+  justify-content: left;
+  text-align: left;
+  margin: 3px;
+  padding: 1% 2% 2% 2%;
+}
+
+//Background
+@mixin Background {
+  background-color: $softwireBlack;
+  width: 100%;
+  height: auto;
+  padding: 0;
+  margin: 0;
+  border-style: none;
+  font-family: $softwireFont;
+
+}
+
+//Buttons
+@mixin Buttons($color, $fontColor)  {
+  width: 45px;
+  height: 10px;
```

Comment:
I'd recommend going with padding here rather than width and height - it means the styling is less likely to have to change if the text in the button changes.